xapian-letor/tests/runsrv.in

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
```

Comment:
This is for running remote tests in -core; since we don't (currently) have a way of packaging the letor data within a database, I doubt this is going to be used in the letor tests.